src/NLog.StructuredLogging.Json.Tests/LoggerExtensionsScopeTests.cs

```diff
@@ -54,6 +54,21 @@ public void TearDown()
         }
 
         [Test]
+        public void Should_Fail_If_Logger_Is_Null()
+        {
+            ILogger nullLogger = null;
+            var exeption = Assert.Throws<ArgumentNullException>(() => nullLogger.BeginScope("should fail"));
```

Comment:
Hmm, it is really strange, but I do not have such overload at all.
`Assert` class is provided by `nunit\3.7.1\lib\net45\nunit.framework.dll`